### ISSUE TITLE
Video quality issue

### ISSUE THREAD

**Issue by iHaiderAli** [kapilmhr/Compressor#1]
Video is compressing but loosing the video quality, so how can I improve the video quality by changing the code?

**Comment by PranotiShinde** [comment]
Video compression taking too much time to compress video

**Comment by kapilmhr** [comment]
Yeah It needs some improvements. I haven't maintained the library now. I will make a new one in kotlin soon and publish it. Thanks
